progress2.inc

```diff
@@ -10,10 +10,10 @@
 #define _progress2_included
 
 #include <a_samp>
-#include <logger>
-#include <YSI_Data\y_iterate>
+#tryinclude <logger>
```

Comment:
there's not much point in making this optional, these are tools to make the library author's life easier - having the library there or not there does not affect API consumers so you may as well just remove this entirely since you already went through the effort to do things manually.

I'd still caution to use tools that prevent subtle bugs though, especially with ID pools (like y_iterate) and ALS hooks. But I won't block this PR if you still want to remove this.

Reply:
I often noticed that ordinary users use either ready-made developments without changes or fixed old versions from dubious sources outside github (in the case when users are not experienced enough and cannot install the necessary libraries themselves).
You can still test the old way. But, if you want to leave these libraries mandatory, then in the future it is worth adding support for dependencies, for example, using github or sampctl.